content/sensu-enterprise/3.0/configuration.md

```diff
@@ -52,4 +52,16 @@ required       | false
 default        | `16384`
 example        | {{< highlight shell >}}MAX_OPEN_FILES=32768{{< /highlight >}}
 
+## Sensu Enterprise command line interfaces and arguments
+
+The following command line options are unique to Sensu Enterprise.
+For command line options shared with `sensu-server`, `sensu-api`, and `sensu-client`,
+visit the [Sensu Core configuration reference docs][1].
+
+-a (-\-api_only) | 
+----------------|------
+description     | Run only the Sensu Enterprise API. You can use API-only mode to deploy Sensu Enterprise API instances that don't process events from transport queues.
```

Comment:
Is it possible to provide more context around why you might want to do this?

Reply:
From @rgeniesse via slack
> re:#477, found some conversations in #enterprise on May 24th. Trying to track down the reason, but it appears customer driven or customer issue driven as Cameron mentions help scout. _continues reading_
appears related to Dashboard performance issues and wanting to scale the API without scaling event processors (the sensu-server portion of sensu-enterprise)

Reply:
@apaskulin @rgeniesse Richard's read is correct, we're adding this feature as one possibility for addressing dashboard latency problems reported by some users, although I suspect code changes in the dashboard (uchiwa and enterprise) are more likely to resolve the issue for customers who are experiencing this slowness.